src/librustdoc/config.rs

```diff
@@ -78,6 +78,8 @@ pub struct Options {
     pub should_test: bool,
     /// List of arguments to pass to the test harness, if running tests.
     pub test_args: Vec<String>,
+    /// Otional path to persist the doctest executables to.
```

Comment:
Typo on "Otional". I'd also want to add something like "If not set, uses a temporary directory."

Reply:
Ah shoot, I thought I checked for those submodules before committing, I'll try to get that stuff fixed tonight.